Adaptors/MongoDB/src/ResultTable.cs

```diff
@@ -379,6 +273,66 @@ public class ResultTable : IResultTable
                           .ConfigureAwait(false);
   }
 
+  /// <inheritdoc />
+  public async Task<Result> UpdateOneResult(string                                                                      sessionId,
```

Comment:
I don't know if it is time we remove it from the interfaces.
Maybe in another PR

Reply:
Yes, I think it is better in another PR.